### DISCUSSION TITLE
Optimism Governor Initial Limits

### DISCUSSION THREAD

**Discussion by claudijd** [wormhole-foundation/wormhole#1942] [Governance] [answered]
This issue can be used to discuss the initial limits for Governor for the Optimism blockchain.  For most chains, they tend to start between $200k/$20k -or- $500k/$50k.

Welcome any feedback from Guardians on which of the two is most appropriate or whether an exception will be made based on anticipation of the new chain.

I'm going to start with a suggestion of $200k/$20k.

**Comment by rmahtani** [comment, marked as answer]
My recommendation would be to start with the $200k/$20k limit. I don't foresee a large volume of wrapped assets being bridged into the Optimism ecosystem.

**Reply by aadam-10** [reply]
this is live as of #1869
